pkg/network/route_cache.go

```diff
@@ -349,3 +353,15 @@ func (n *netlinkRouter) getInterface(srcAddress util.Address, srcIP net.IP, netn
 	routeCacheTelemetry.ifCacheSize.Inc()
 	return iff
 }
+
+// CounterIncWithTag adds error tag and increments counter
+func CounterIncWithTag(counter telemetry.Counter, err error) (errno syscall.Errno, ok bool) {
```

Comment:
Does this need to be exported?

Reply:
No it does not... I misinterpreted one of your previous comments